src/main/resources/swagger/api-docs.yaml

```diff
@@ -139,6 +147,45 @@ definitions:
         type: integer
         description: line number
 
+  Cluster:
+    description: ''
+    required:
+      - clusterId
+      - clusterName
+      - googleProject
+      - googleServiceAccount
+      - googleBucket
+      - operationName
+      - createdDate
+      - labels
+    properties:
+      clusterId:
+        type: string
+        description: Google's UUID for the cluster
```

Comment:
Maybe we should we indicate in the name that it's google's ID and call it googleClusterId?